src/environments/environment.ts

```diff
@@ -12,12 +12,13 @@ export const environment = {
     'en-US',
     'fr-FR'
   ],
+
   firebase: {
-    apiKey: 'AIzaSyCzy5B2_jDI7IuYEtrMH00_zkKFoDTu2-k',
-    authDomain: 'ptos-e178e.firebaseapp.com',
-    databaseURL: 'https://ptos-e178e.firebaseio.com/',
-    projectId: 'ptos-e178e',
-    storageBucket: 'ptos-e178e.appspot.com',
-    messagingSenderId: '714481396961'
-  }
+    apiKey: "AIzaSyA5CLPvp0CMpNePd6OHmSpqUSMKErG9tiI",
```

Comment:
@BaidaaAlbzoor  please revert this change, we will be using ptos-e178e firebase app